### PR TITLE
Sync/update to neic

### PR DESCRIPTION
**Pull request by blankdots**

### CODE REVIEW

File: sda-orch/templates/podsecuritypolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - min: 1
+      max: 65535
+  hostNetwork: false
+  hostIPC: false
+  hostPID: false
+  volumes:
+  - secret
+  - configMap
+  - emptyDir
+  - projected
```
Comment (vvaltia) [inline sda-orch/templates/podsecuritypolicy.yaml:39]: Are `configMap` and `projected` needed?

Comment (blankdots) [reply sda-orch/templates/podsecuritypolicy.yaml:39]: good point

File: sda-svc/templates/ingest-deploy.yaml [hunk 2]
```diff
@@ -64,81 +67,149 @@ spec:
...
+        - name: DB_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-ingest
+                key: dbPassword
+        - name: DB_USER
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-ingest
+                key: dbUser
```
Comment (vvaltia) [inline sda-svc/templates/ingest-deploy.yaml:200]: Some indentations not matching up with other lines

File: sda-svc/templates/ingest-deploy.yaml [hunk 3]
```diff
@@ -148,28 +219,32 @@ spec:
...
-    {{- end }}
+    {{- if not .Values.global.secretsService }}
+        - name: c4gh
+          secret:
+            defaultMode: 0440
+            secretName: {{ template "sda.fullname" . }}-c4gh
+            items:
+            - key: {{ .Values.global.c4gh.file }}
+              path: {{ .Values.global.c4gh.file }}
+      {{- end }}
```
Comment (vvaltia) [inline sda-svc/templates/ingest-deploy.yaml:247]: Indentation of end not matching up with if

File: sda-svc/templates/intercept-deploy.yaml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  {{- if not .Values.global.pkiService }}
+        volumeMounts:
+        - name: tls
+          mountPath: {{ template "tlsPath" . }}
+      volumes:
+        - name: {{ "tls"  }}
+          secret:
+            defaultMode: 0440
+            secretName: {{ template "sda.fullname" . }}-intercept-certs
+    {{- end }}
```
Comment (vvaltia) [inline sda-svc/templates/intercept-deploy.yaml:111]: Indentation of end not matching up with if

File: sda-svc/templates/verify-deploy.yaml [hunk 5]
```diff
@@ -64,106 +67,156 @@ spec:
...
+        - name: BROKER_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-verify
+                key: mqPassword
+        - name: BROKER_USER
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-verify
+                key: mqUser
```
Comment (vvaltia) [inline sda-svc/templates/verify-deploy.yaml:174]: Indentation not matching the following indenration

File: sda-svc/templates/verify-deploy.yaml [hunk 6]
```diff
@@ -64,106 +67,156 @@ spec:
...
+        - name: DB_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-verify
+                key: dbPassword
+        - name: DB_USER
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-verify
+                key: dbUser
```
Comment (vvaltia) [inline sda-svc/templates/verify-deploy.yaml:189]: Unconsistent indentation

File: sda-db/templates/podsecuritypolicy.yaml [hunk 7]
```diff
@@ -30,14 +30,39 @@ spec:
...
-  hostNetwork: false
-  hostIPC: false
-  hostPID: false
   volumes:
   - secret
   - persistentVolumeClaim
   - configMap
   - emptyDir
   - projected
   - nfs
```
Comment (vvaltia) [inline sda-db/templates/podsecuritypolicy.yaml:39]: I think only `projected`, `persistentVolumeClaim` and `emptyDir` are used by db statefulset

**Review by vvaltia — APPROVED** [review]

**Pull request merged**